src/framebuffer.ml

```diff
@@ -1,5 +1,8 @@
 
-type t = int array array
+type t = {
+  buffer : int array array;
+  mutable dirty : bool;
```

Comment:
The dirty bit looks good to me, but is it actually being used anywhere? I thought the idea was for the dirty bit to help with not re-rendering a framebuffer if nothing has changed between two ticks?